st2client/st2client/commands/pack.py

```diff
@@ -210,7 +210,10 @@ def run(self, args, **kwargs):
         if not is_structured_output:
             self._get_content_counts_for_pack(args, **kwargs)
 
-        return self.manager.install(args.packs, python3=args.python3, force=args.force,
+        if args.python3:
+            print('\nDEPRECATION WARNING: --python3 flag is ignored and will be removed in v3.5.0 as StackStorm now runs with python3 only\n')
```

Comment:
Any reason to not use [`warnings.warn`](https://docs.python.org/3/library/warnings.html#warnings.warn) here?